roles/ceph-osd/tasks/system_tuning.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: disable osd directory parsing by updatedb
+  command: updatedb -e /var/lib/ceph
+  changed_when: false
+  failed_when: false
+
+- name: disable transparent hugepage
+  command: "echo never > /sys/kernel/mm/transparent_hugepage/enabled"
+  changed_when: false
+  failed_when: false
+  when: ceph_osd.disable_transparent_hugepage
+
+- name: disable swap
+  command: swapoff -a
+  changed_when: false
+  failed_when: false
+  when: ceph_osd.disable_swap
```

Comment:
Is there a way to remove the swap line from fstab so that this survives reboots?